extension/app/inpage/ts/inpage.ts

```diff
@@ -391,6 +405,24 @@ class InterceptorMessageListener {
 		this.sendMessageToBackgroundPage({ method: 'connected_to_signer', params: [signerName] })
 	}
 
+	private readonly injectUnSupportedMethods = (windowEthereum: WindowEthereum & UnSupportedWindowEthereumMethods) => {
+		const unSupportedError = (method: string) => {
```

Comment:
`unsupported` is one word.
```suggestion
		const unsupportedError = (method: string) => {
```